server/src/database.js

```diff
@@ -35,11 +64,14 @@ const createStore = () => {
         email: {
           type: Sequelize.STRING, allowNull: false,
         },
-        token: Sequelize.STRING,
         birthday: Sequelize.DATE,
         city: Sequelize.STRING,
         state: Sequelize.STRING,
+        zip: Sequelize.STRING,
+        additionalAddress: Sequelize.STRING,
```

Comment:
Should we just name this as address? Typically the address fields in US can be stored as:

- street address
- additional street address
- city
- state
- zip code

If we don't want to split into all these fields, we can just store everything into "address" field for now.

Reply:
Modified! Thank you.